nbs/docs/tutorials/ConformalPrediction.ipynb

```diff
@@ -0,0 +1,14030 @@
+{
```

Comment:
<p>In the <a href="https://nixtla.github.io/statsforecast/docs/tutorials/conformalprediction.html" target="_blank">current version </a> there's a problem displaying the plots. </p><p> </p><p><code>Unable to display output for mime type(s): application/vnd.plotly.v1+json</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/597/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>